aether-kernel/aether/kernel/api/entity_extractor.py

```diff
@@ -120,7 +120,11 @@ def extract_view(request, *args, **kwargs):
 
     delta = request.query_params.get('delta', '1d')
     modified = parse_delta(delta)
-    submissions = Submission.objects.filter(modified__lte=modified).filter(is_extracted=False)
+    submissions = (
+        Submission.objects
+        .filter(modified__lte=modified)
+        .filter(is_extracted=False)
```

Comment:
```suggestion
        .filter(is_extracted=False)
        .order_by('-modified')
```

With this small change you force the `modified` index